pytorch_lightning/utilities/imports.py

```diff
@@ -30,33 +31,32 @@ def _module_available(module_path: str) -> bool:
     """
     # todo: find a better way than try / except
```

Comment:
@Borda I don't think there is a better way. The Python documentation clearly indicates that AttributeError/ModuleNotFoundError will be raised: https://docs.python.org/3/library/importlib.html#importlib.util.find_spec

Reply:
Given your :+1:, i'll remove the TODO again

```suggestion
```